pyweather.py

```diff
@@ -1909,7 +2001,7 @@ def radar_clearImages():
     summary_forecastIterations = summary_forecastIterations + 1
     if summary_forecastIterations == 5:
         break
-print("")

```

Comment:
Please do not delete this print statement, it creates whitespace in the terminal.